mrchecker-framework-modules/mrchecker-webapi-module/src/test/java/com/capgemini/mrchecker/webapi/httpbin/HtmlTest.java

```diff
@@ -0,0 +1,40 @@
+package com.capgemini.mrchecker.webapi.httpbin;
+
+import com.capgemini.mrchecker.test.core.logger.BFLogger;
+import com.capgemini.mrchecker.webapi.BasePageWebApiTest;
+import com.capgemini.mrchecker.webapi.pages.httbin.HtmlPage;
+import io.restassured.response.Response;
+import io.restassured.response.ResponseBody;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.select.Elements;
+import org.junit.Test;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+public class HtmlTest extends BasePageWebApiTest {
+	private HtmlPage htmlPage = new HtmlPage();
+
+	@Test
+	public void getHtmlPage() {
```

Comment:
Can you see any other test case that you can put here?

Reply:
This endpoint gives always the same page in response. I can't see any other checks.